### ISSUE TITLE
Yelp api returns different result count than the yelp.com

### ISSUE THREAD

**Issue by leonidz83** [Yelp/yelp-fusion#493]
## Overview
- **Client ID:** qgjcHsI-weygz6W836-94Q
- **Issue type:** bug
- **Summary:** Number of records returned from api is not the same as using directly via yelp.com
- **Platform:** Desktop

## Description
It seems that the API and Yelp.com does not provide the same result set for the same search criteria 

## More information
Web - number of results 510
`https://www.yelp.ca/search?find_desc=Laser%20Hair%20Removal&find_loc=Mississauga%2C%20On`

Api - number or results 321 (limit was set to 1 to fetch the total)
`search_results = yelp_api.search_query(term='Laser Hair Removal', location='Mississauga, On', limit=1, offset=offset)`

### Endpoint
YelpAPI

### Parameters or Sample Request

`search_results = yelp_api.search_query(term='Laser Hair Removal', location='Mississauga, On', limit=1, offset=offset)`

### Response
`{
	'total': 321,
	'businesses': [{
		'review_count': 11,
		'url': 'https: //www.yelp.com/biz/the-body-clinic-day-spa-mississauga?adjust_creative=qgjcHsI-weygz6W836-94Q&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=qgjcHsI-weygz6W836-94Q',
		'distance': 6198.395208081101,
		'coordinates': {
			'latitude': 43.542250258203,
			'longitude': -79.6263379631581
		},
		'categories': [{
			'title': 'DaySpas',
			'alias': 'spas'
		},
		{
			'title': 'SkinCare',
			'alias': 'skincare'
		},
		{
			'title': 'LaserHairRemoval',
			'alias': 'laser_hair_removal'
		}],
		'id': 'Qx5dhODc6vKib_99IzxmCw',
		'phone': '+19052712980',
		'is_closed': False,
		'location': {
			'country': 'CA',
			'address2': 'Unit2',
			'zip_code': 'L5H4C7',
			'display_address': ['1200VanierDrive',
			'Unit2',
			'Mississauga,
			ONL5H4C7',
			'Canada'],
			'address3': '',
			'city': 'Mississauga',
			'state': 'ON',
			'address1': '1200VanierDrive'
		},
		'display_phone': '+1905-271-2980',
		'alias': 'the-body-clinic-day-spa-mississauga',
		'transactions': [],
		'image_url': 'https: //s3-media3.fl.yelpcdn.com/bphoto/J8rqd4Zelwi-Mn66dIC1Ag/o.jpg',
		'price': '$$',
		'name': 'TheBodyClinicDaySpa',
		'rating': 5.0
	}],
	'region': {
		'center': {
			'latitude': 43.59313723570342,
			'longitude': -79.65774536132812
		}
	}
}`

## Extra information
_(Anything else you that you think might help us better resolve this)_

**Comment by dan98765** [comment]
Hi @leonidz83 - the business search endpoint accept parameters that adjust how search works more exactly than you can do on yelp.com. For example, `radius` is one parameter you can specify via API that has a big impact on the area of the search executed. When searching on Yelp's www or mobile page, the radius of the search is dynamically adjusted behind-the-scenes. Filtering by category is another thing that the www search may try to silently 'guess' that API search instead gives developers explicit control over. 

Long story short, it is very unlikely a search using the API and a search on yelp.com will return exactly the same number of results, so we do not treat it as a bug. 

What we do investigate & debug is when there are some high-relevance businesses that yelp.com search returns but are not returned when searching via API regardless of what parameters/filters are used. If you encounter situations like this, please open an issue with the specifics so we can dig into it.
